From 39ce9fb76c4c0838fde4df3362614eb0eea5fb6b Mon Sep 17 00:00:00 2001
From: Chanho Park <chanho61.park@samsung.com>
Date: Fri, 22 Nov 2013 19:11:44 +0900
Subject: [PATCH 1026/1302] tizen: update default config to use slp gadget

The Tizen uses slp composite gadget which is based on android gadget.
Until we migrate to configfs based gadget configuration, we should use the slp
gadget. It only support sdb and rndis gadget.

Change-Id: I9e7afd3ef9865d43d43d2221b30b7b10ef039399
Signed-off-by: Chanho Park <chanho61.park@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/configs/tizen_defconfig | 33 +++++++++++++++++++++------------
 1 file changed, 21 insertions(+), 12 deletions(-)

diff --git a/arch/arm/configs/tizen_defconfig b/arch/arm/configs/tizen_defconfig
index a6559b1..7f0e7ea 100644
--- a/arch/arm/configs/tizen_defconfig
+++ b/arch/arm/configs/tizen_defconfig
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/arm 3.10.0 Kernel Configuration
+# Linux/arm 3.10.19 Kernel Configuration
 #
 CONFIG_ARM=y
 CONFIG_ARM_HAS_SG_CHAIN=y
@@ -15,6 +15,7 @@ CONFIG_LOCKDEP_SUPPORT=y
 CONFIG_TRACE_IRQFLAGS_SUPPORT=y
 CONFIG_RWSEM_GENERIC_SPINLOCK=y
 CONFIG_ARCH_HAS_CPUFREQ=y
+CONFIG_ARCH_HAS_BANDGAP=y
 CONFIG_GENERIC_HWEIGHT=y
 CONFIG_GENERIC_CALIBRATE_DELAY=y
 CONFIG_NEED_DMA_MAP_STATE=y
@@ -406,6 +407,7 @@ CONFIG_SWP_EMULATE=y
 # CONFIG_CPU_ICACHE_DISABLE is not set
 # CONFIG_CPU_DCACHE_DISABLE is not set
 # CONFIG_CPU_BPREDICT_DISABLE is not set
+CONFIG_KUSER_HELPERS=y
 CONFIG_OUTER_CACHE=y
 CONFIG_OUTER_CACHE_SYNC=y
 CONFIG_MIGHT_HAVE_CACHE_L2X0=y
@@ -543,11 +545,13 @@ CONFIG_CPU_FREQ_STAT=y
 # CONFIG_CPU_FREQ_DEFAULT_GOV_USERSPACE is not set
 CONFIG_CPU_FREQ_DEFAULT_GOV_ONDEMAND=y
 # CONFIG_CPU_FREQ_DEFAULT_GOV_CONSERVATIVE is not set
+# CONFIG_CPU_FREQ_DEFAULT_GOV_LAB is not set
 CONFIG_CPU_FREQ_GOV_PERFORMANCE=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_CPU_FREQ_GOV_ONDEMAND=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
+# CONFIG_CPU_FREQ_GOV_LAB is not set
 
 #
 # ARM CPU frequency scaling drivers
@@ -580,8 +584,7 @@ CONFIG_BINFMT_ELF=y
 CONFIG_ARCH_BINFMT_ELF_RANDOMIZE_PIE=y
 CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS=y
 CONFIG_BINFMT_SCRIPT=y
-CONFIG_HAVE_AOUT=y
-# CONFIG_BINFMT_AOUT is not set
+# CONFIG_HAVE_AOUT is not set
 # CONFIG_BINFMT_MISC is not set
 CONFIG_COREDUMP=y
 
@@ -1864,6 +1867,16 @@ CONFIG_THERMAL_GOV_STEP_WISE=y
 # CONFIG_THERMAL_GOV_USER_SPACE is not set
 CONFIG_CPU_THERMAL=y
 # CONFIG_THERMAL_EMULATION is not set
+# CONFIG_IMX_THERMAL is not set
+
+#
+# Texas Instruments thermal drivers
+#
+# CONFIG_TI_SOC_THERMAL is not set
+
+#
+# Samsung thermal drivers
+#
 CONFIG_EXYNOS_THERMAL=y
 CONFIG_WATCHDOG=y
 CONFIG_WATCHDOG_CORE=y
@@ -2562,17 +2575,12 @@ CONFIG_USB_S3C_HSOTG=y
 # CONFIG_USB_NET2272 is not set
 # CONFIG_USB_DUMMY_HCD is not set
 CONFIG_USB_LIBCOMPOSITE=y
-CONFIG_USB_U_ETHER=y
-CONFIG_USB_U_RNDIS=y
-CONFIG_USB_F_ECM=y
-CONFIG_USB_F_SUBSET=y
-CONFIG_USB_F_RNDIS=y
+CONFIG_USB_F_ACM=y
+CONFIG_USB_U_SERIAL=y
 # CONFIG_USB_CONFIGFS is not set
 # CONFIG_USB_ZERO is not set
 # CONFIG_USB_AUDIO is not set
-CONFIG_USB_ETH=y
-CONFIG_USB_ETH_RNDIS=y
-# CONFIG_USB_ETH_EEM is not set
+# CONFIG_USB_ETH is not set
 # CONFIG_USB_G_NCM is not set
 # CONFIG_USB_GADGETFS is not set
 # CONFIG_USB_FUNCTIONFS is not set
@@ -2580,7 +2588,7 @@ CONFIG_USB_ETH_RNDIS=y
 # CONFIG_USB_G_SERIAL is not set
 # CONFIG_USB_MIDI_GADGET is not set
 # CONFIG_USB_G_PRINTER is not set
-# CONFIG_USB_G_SLP is not set
+CONFIG_USB_G_SLP=y
 # CONFIG_USB_CDC_COMPOSITE is not set
 # CONFIG_USB_G_NOKIA is not set
 # CONFIG_USB_G_ACM_MS is not set
@@ -3084,6 +3092,7 @@ CONFIG_CM36651=y
 CONFIG_AK8975=y
 # CONFIG_IIO_ST_MAGN_3AXIS is not set
 CONFIG_PWM=y
+CONFIG_PWM_SYSFS=y
 CONFIG_PWM_SAMSUNG=y
 CONFIG_IRQCHIP=y
 CONFIG_ARM_GIC=y
-- 
1.8.3.2

